test/bats/vault.bats

```diff
@@ -86,17 +86,21 @@ EOF
   assert_success
 
   run kubectl exec -ti $VAULT_POD -- vault policy write example-readonly -<<EOF
+path "sys/mounts" {
```

Comment:
Is this a new policy required starting from `0.0.5` release of the vault provider?

Reply:
Sorry I missed this before. Yes, that's right, it does a read to check whether the `kv` engine is v1 or v2 starting from 0.0.5